package.json

```diff
@@ -28,6 +28,7 @@
     "@matterlabs": "github:matter-labs/era-contracts",
     "@openzeppelin/contracts-upgradeable": "v5.0.1",
     "@rhinestone/modulekit": "github:rhinestonewtf/modulekit#test/safe-latest",
+    "@safe-global/safe-contracts": "1.4.1-build.0",
```

Comment:
Why did the safe contracts get added here? We can actually pull them in without doing this

See [remappings](https://github.com/zkemail/email-recovery/blob/4e7031693d8e97cfcbc42b7d063a748a0a53b952/remappings.txt#L13)
See [example](https://github.com/zkemail/email-recovery/blob/4e7031693d8e97cfcbc42b7d063a748a0a53b952/test/integration/SafeRecovery/SafeRecovery.t.sol#L8)